.stylelintrc.json

```diff
@@ -0,0 +1,10 @@
+{
+    "extends": ["stylelint-config-standard"],
+    "plugins": ["stylelint-scss", "stylelint-csstree-validator"],
+    "rules": {
+      "at-rule-no-unknown": null,
+      "scss/at-rule-no-unknown": true,
+      "csstree/validator": true
+    },
+    "ignoreFiles": ["build/**", "dist/**", "**/reset*.css", "**/bootstrap*.css"]
+  }
```

Comment:
Great job so far 👋👋👋.

- You adhere to the Github flow.👊👊
- You follow the semantic HTML and CSS rule👌👌
However, your stylelintrc.jason was not configured properly to detect bugs in your code😪😪.
i will kindly suggest you configure them well to have an error-free code🤗🤗